packages/terra-date-picker/lib/DatePicker.scss

```diff
@@ -0,0 +1,54 @@
+@import '~terra-mixins';
+@import '../node_modules/react-datepicker/dist/react-datepicker.min';
+
+.terra-DatePicker {
+  display: inline-block;
+  min-width: 14.643em; 
+}
+
+.terra-DatePicker-input {
+  border: 0.071em solid $terra-grey-40;
+  box-shadow: inset 0 0.143em 0.143em $terra-grey-20;
+  font-size: 1em;
+  height: 1.143em;
+  line-height: 1.143em;
```

Comment:
This came up a few weeks ago, line height should be unitless. 

Here is a short article Matt Butler posted on a different pull request: https://css-tricks.com/almanac/properties/l/line-height/#article-header-id-0